chromium_src/chrome/browser/net/stub_resolver_config_reader.cc

```diff
@@ -0,0 +1,41 @@
+/* Copyright (c) 2021 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "base/strings/strcat.h"
```

Comment:
Only needed #if BUILDFLAG(UNSTOPPABLE_DOMAINS_ENABLED)

Reply:
Fixed in https://github.com/brave/brave-core/pull/7702/commits/eda54f0951b148bab82d78245e0f7ad292218853